src/openagi/memory/storage.py

```diff
@@ -0,0 +1,38 @@
+import chromadb
+
+client = chromadb.Client()
```

Comment:
create a separate storage class.
Do not initialize chroma globally, initialize it at once in the beginning and use the same instance.